packages/core/src/prompt.ts

```diff
@@ -21,67 +23,54 @@ interface Question {
  */
 export async function promptConfirmation(message: string): Promise<boolean> {
   log.pause();
-  const answers = await inquirer.prompt([
-    {
-      type: 'expand',
-      name: 'confirm',
-      message,
-      default: 2, // default to help in order to avoid clicking straight through
-      choices: [
-        { key: 'y', name: 'Yes', value: true },
-        { key: 'n', name: 'No', value: false },
-      ],
-    },
-  ]);
+  const answers = await expand({
+    message,
+    default: 'h', // default to help in order to avoid clicking straight through
+    choices: [
+      { key: 'y', name: 'Yes', value: true },
+      { key: 'n', name: 'No', value: false },
+    ],
+  });
   log.resume();
 
-  return answers.confirm;
+  return answers;
 }
 
 /**
  * Prompt for selection
  * @param {string} message
- * @param {{ choices: import("inquirer").ListChoiceOptions[] } & Pick<import("inquirer").Question, 'filter' | 'validate'>} [options]
+ * @param {Question} [options]
  * @returns {Promise<string>}
  */
-export async function promptSelectOne(message: string, { choices, filter, validate } = {} as Question): Promise<string> {
+export async function promptSelectOne(message: string, { choices }: Question = {} as Question): Promise<string> {
```

Comment:
```suggestion
export async function promptSelectOne(
  message: string,
  { choices }: Partial<Parameters<typeof select>[0]> = {}
): Promise<string[]> {
```

With `as` you're ignoring type check, and you can see if breaks the typing since it allow you to return `Promise<string>` while `select` returns an array, not a string 😬